server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -471,8 +471,12 @@ export function configureWebSocketServices(
 						for (const signal of contentBatches) {
 							if (isSentSignalMessage(signal)) {
 								const signalMessage: ISignalMessage = {
-									...signal,
 									clientId,
+									content: signal.content,
+									clientConnectionNumber: signal.clientConnectionNumber,
+									referenceSequenceNumber: signal.referenceSequenceNumber,
+									type: signal.type,
+									targetClientId: signal.targetClientId,
```

Comment:
I would recommend going back to the spread approach. Being so precise limits flexibility. If we wanted to add other pass-thru properties, we'll have to carefully stage.

Reply:
I changed it because I found through testing that the spread approach made it so additional properties can be passed through and inserted in the return signal. Is there a way to reconcile this while also doing `..signal`?

Reply:
e.g., if there was signal.additionalProperty = "additional property", it would be included in the ISignalMessage

Reply:
You could use ... and restrict by deleting, but I am suggesting that we don't restrict. I don't see the upside to restricting pass-thru. I can see it causing extra work later.